content/en/agent/faq/kubernetes-state-cluster-check.md

```diff
@@ -0,0 +1,102 @@
+---
+title: How do I run the kubernetes_state check as a cluster check?
+kind: faq
+further_reading:
+- link: "agent/kubernetes"
+  tag: "Documentation"
+  text: "Kubernetes with Datadog"
+- link: "agent/autodiscovery"
+  tag: "Documentation"
+  text: "Using Autodiscovery with Kubernetes and Docker"
+- link: "agent/autodiscovery/clusterchecks"
+  tag: "Documentation"
+  text: "Running Cluster Checks with Autodiscovery"
+---
+
+The `kubernetes_state` check, which monitors `kube-state-metrics`, is enabled automatically by [Autodiscovery][1]. In larger Kubernetes clusters, this check may consume too much RAM. In this case, consider running the `kubernetes_state` check as a [cluster check][2].
+
+## Configuration
+
+1. Deploy `kube-state-metrics` with cluster check annotations on the service:
+
+	```
+	ad.datadoghq.com/service.check_names: '["kubernetes_state"]'
+	ad.datadoghq.com/service.init_configs: '[{}]'
+	ad.datadoghq.com/service.instances: |
+	    [
+	    {
+	        "kube_state_url": "http://%%host%%:8080/metrics",
+	        "prometheus_timeout": 30,
```

Comment:
the timeout and min collection interval are set to 30 here because we have very large payloads, but it's worth a disclaimer as it means they would only collect their metrics at a lower granularity.
I would suggest mentioning them or pointing to the doc for the ksm check, but no need to lower the defautl granularity without explanation.

Reply:
agreed, we can remove this line and the next by default, and maybe add a section to the faq about the check timing out and the need to decrease the frequency and increasing the timeout.